types/fundamental-react/lib/Select/Select.d.ts

```diff
@@ -1,21 +1,33 @@
-import * as React from "react";
+import * as React from 'react';
 
 export interface SelectProps {
```

Comment:
Additional props
```ts
popoverProps?: Record<string, any>;
validationOverlayProps?: {
  className?: string;
  formMessageProps?: Record<string, any>;
  innerRefClassName?: string;
  popperClassName?: string;
  referenceClassName?: string;
  wrapperProps?: Record<string, any>;
}
```

Reply:
thanks for the addons, i will add them

Reply:
Did I miss something?  These should have been included right?

popoverProps: https://github.com/SAP/fundamental-react/blob/master/src/Select/Select.js#L258

validationOverlayProps: https://github.com/SAP/fundamental-react/blob/master/src/Select/Select.js#L268